### PR TITLE
Split fixture data and persistance 

### PR DESCRIPTION
**Pull request by mmadariaga**
Separate entity instatiation from fixtures so that entities can be used on unit tests as well

#### Type Of Change 
- [ ] Small bug fix
- [X] New feature or enhancement
- [ ] Breaking change

#### Checklist:
- [X] Commits are named and have tag following [commit rules](https://github.com/irontec/ivozprovider/blob/bleeding/doc/dev/en/commits.md)
- [X] Commits are split per component (schema, web/admin, kamusers, agis, ..)
- [ ] Changes have been tested locally
- [ ] Fixes an existing issue (Fixes #XXXX)

### CODE REVIEW

**Comment by ironArt3mis** [comment]
Rebase completed.

File: library/DataFixtures/Stub/Ast/PsEndpointStub.php [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+            ->setDirectMediaMethod('invite')
+            ->setOutboundProxy('sip:users.ivozprovider.local^3Blr')
+            ->setTrustIdInbound('yes')
+            ->setFriendId(
+                1
+            );
+        $this->append($dto);
+
+        $dto = (new PsEndpointDto(5))
+            ->setSorceryId('b1c1f1_testResidential')
```
Comment (Kaian) [inline library/DataFixtures/Stub/Ast/PsEndpointStub.php:75]: Not sure if this is relevant but I think the correct sorceryId would be b1c1r5_testResidential
Same goes for Aors field.

File: library/DataFixtures/Stub/Ast/PsEndpointStub.php [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+            ->setFromDomain('127.0.0.1')
+            ->setAors('b1c1f1_testResidential')
+            ->setContext('friends')
+            ->setAllow('alaw')
+            ->setDirectMediaMethod('invite')
+            ->setOutboundProxy('sip:users.ivozprovider.local^3Blr')
+            ->setTrustIdInbound('yes')
+            ->setMailboxes('residential2@company1')
```
Comment (Kaian) [inline library/DataFixtures/Stub/Ast/PsEndpointStub.php:83]: Again, the correct mailbox would be residential1@company1

File: library/DataFixtures/Stub/Ast/VoicemailStub.php [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+        $dto = (new VoicemailDto(3))
+            ->setContext('company1')
+            ->setMailbox('101')
+            ->setFullname('Alice ')
+            ->setEmail('alice@democompany.com')
+            ->setAttach('yes')
+            ->setTz('Europe/Madrid')
+            ->setUserId(
+                1
+            );
```
Comment (Kaian) [inline library/DataFixtures/Stub/Ast/VoicemailStub.php:53]: This should be actually an error. UserId should be unique in ast_voicemail.

File: library/DataFixtures/Stub/StubTrait.php [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+    {
+        $entity = $this->entityAssembler->createFromDto(
+            $dto,
+            $this->getEntityName()
+        );
+        $entityId = $entity->getId();
+
+        if (array_key_exists($entityId, $this->items)) {
+            throw new \Exception(
+                $this->getEntityName() . ' #' . $entityId . ' already exists'
```
Comment (Kaian) [inline library/DataFixtures/Stub/StubTrait.php:35]: This looks like what entities do in toString method

File: library/DataFixtures/Stub/Ast/PsEndpointStub.php [hunk 5]
```diff
@@ -0,0 +1,89 @@
...
+            ->setFriendId(
+                1
+            );
+        $this->append($dto);
+
+        $dto = (new PsEndpointDto(5))
+            ->setSorceryId('b1c1f1_testResidential')
+            ->setFromDomain('127.0.0.1')
+            ->setAors('b1c1f1_testResidential')
+            ->setContext('friends')
```
Comment (Kaian) [inline library/DataFixtures/Stub/Ast/PsEndpointStub.php:78]: residential here

File: library/DataFixtures/Stub/Ast/VoicemailStub.php (hunk 3 above)
Comment (mmadariaga) [reply library/DataFixtures/Stub/Ast/VoicemailStub.php:53]: I'll keep this until we add unique constraint

**Pull request merged**